packages/accordions/src/styled/accordion/StyledPanel.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+export interface IStyledPanel {
+  isBare?: boolean;
+  isCompact?: boolean;
+  isExpanded?: boolean;
+}
+
+const COMPONENT_ID = 'accordions.panel';
+
+const paddingStyles = (props: IStyledPanel & ThemeProps<DefaultTheme>) => {
+  const { base } = props.theme.space;
+  let paddingTop = base * 2;
+  let paddingHorizontal = base * 5;
+  let paddingBottom = base * 8;
+
+  if (props.isCompact) {
+    paddingTop = base * 2;
+    paddingHorizontal = base * 3;
+    paddingBottom = base * 4;
+  }
+
+  if (props.isExpanded === false) {
+    paddingTop = 0;
+    paddingBottom = 0;
+  }
+
+  return css`
+    transition: padding 0.25s ease-in-out;
+    padding: ${paddingTop}px ${paddingHorizontal}px ${paddingBottom}px;
+  `;
+};
+
+export const StyledPanel = styled.div.attrs<IStyledPanel>({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledPanel>`
+  ${paddingStyles};
+  border-bottom: ${props =>
```

Comment:
Do we want to programmatically add a top border to `first-child` by any chance? It seems a smidge odd to me visually that the first list item doesn't have a border but the last has one on the bottom. https://share.getcloudapp.com/NQuDZQl5

I think I'd advocate for ditching the last border or adding a top border.

Reply:
@rossmoody @hzhu great point, i'm for ditching the border for `last-child`!

Reply:
Update: removed the border on the last panel of the `Accordion`.